process/smartContract/process.go

```diff
@@ -1422,8 +1484,9 @@ func (sc *scProcessor) createSCRsWhenError(
 	}
 
 	accumulatedSCRData := ""
-	if sc.isCrossShardESDTTransfer(tx) {
-		accumulatedSCRData += string(tx.GetData())
+	esdtReturnData, isCrossShardESDTCall := sc.isCrossShardESDTTransfer(tx)
+	if callType != vmcommon.AsynchronousCallBack && isCrossShardESDTCall {
```

Comment:
Is this a case of backwards compatibility?

Reply:
yes it is. this code was not enabled yet.